code/game/movietitles.dm

```diff
@@ -124,8 +124,8 @@ GLOBAL_LIST(end_titles)
 		if(!cast.len && !chunksize)
 			chunk += "CAST:"
 		var/job = ""
-		if(GetAssignment(H) != "Unassigned")
-			job = ", [uppertext(GetAssignment(H))]"
+		if(H.get_assignment("NONE", "NONE") != "NONE")
```

Comment:
get rid of the global GetAssignment proc all together if you're gonna replace all of the use cases.

Reply:
nevermind im fucking blind apparently